_includes/tell-me-why/manage-notifications.md

```diff
@@ -0,0 +1,33 @@
+##Managing Notifications
+Once you've commented on an _issue_ or _pull request_, email notifications will start to pour in. By default notifications will be send by email when there is activity in the thread.  
+You can also choose to receive notifications of a full repository. Whenever there is a new _issue_, _pull request_ or _comment_ posted you will get a notification.
+
+###You can silence or unmute notifications of _individual threads_.
```

Comment:
👍  I wasn't sure about this. I initially settled for short, but see that explicitly mentioning _"issue"_ and _"pull request"_ might be less confusing.